app/renderer/js/utils/logger-util.js

```diff
@@ -5,10 +5,23 @@ const { initSetUp } = require('./default-util');
 const { sentryInit, captureException } = require('./sentry-util');
 
 initSetUp();
-sentryInit();
+
 let app = null;
+let reportErrors = true;
 if (process.type === 'renderer') {
 	app = require('electron').remote.app;
+
+	// Report Errors to Sentry only if it is enabled in settings
+	// Gets the value of reportErrors from config-util for renderer process
+	// For main process, sentryInit() is handled in index.js
+	const { ipcRenderer } = require('electron');
+	ipcRenderer.send('error-reporting');
```

Comment:
I forgot why we are using this method instead of getting the value from configUtil?

Reply:
This was done because directly using ConfigUtil here causes a circular dependency (or say a race condition) between the ConfigUtil and the LoggerUtil.